doc/source/_templates/defindex.html

```diff
@@ -1,35 +0,0 @@
-{#
-    basic/defindex.html
-    ~~~~~~~~~~~~~~~~~~~
-
-    Default template for the "index" page.
-
-    :copyright: Copyright 2007-2017 by the Sphinx team, see AUTHORS.
-    :license: BSD, see LICENSE for details.
```

Comment:
I don't think the sentence "Welcome! This is the documentation for NumPy ..., last updated ..." can be copyrighted, so deleting this message should be fine.